packages/mds-policy-service/service/helpers.ts

```diff
@@ -0,0 +1,30 @@
+import { uuid } from '@mds-core/mds-utils'
+import type { PolicyDomainModel } from '../@types'
+import type {
+  BaseIntentPolicyUserFields,
+  BaseIntentRuleUserFields,
+  IntentDraft,
+  INTENT_TYPE,
+  NoParkingIntentPolicyUserFields,
+  NoParkingIntentRuleUserFields
+} from '../@types/intents'
+import { BASE_POLICY_DEFAULTS, INTENT_RULE_CONSTANTS } from '../@types/intents'
+
+export function translateIntentToPolicy<
+  I extends INTENT_TYPE = 'no_parking',
```

Comment:
We may not always have a clean division between policy fields and rule fields in policy intents, so I suspect that we may run into trouble here with the type inferencing and structure of this function in the future but we shall see.